tests/end_to_end/end_to_end.cpp

```diff
@@ -139,6 +160,51 @@ typedef class _udp_packet
         return _packet;
     }
 
+    void
+    set_mac_addresses(_In_ const std::array<uint8_t, 6>& source_mac, _In_ const std::array<uint8_t, 6>& destination_mac)
+    {
+        auto eth = reinterpret_cast<ebpf::ETHERNET_HEADER*>(_packet.data());
```

Comment:
```suggestion
        auto ethernet_header = reinterpret_cast<ebpf::ETHERNET_HEADER*>(_packet.data());
```
style guide says to avoid abbreviations.  Same in 173, 182.